assembl/alembic/versions/b0871ff12f95_thematic_identifier.py

```diff
@@ -0,0 +1,26 @@
+"""thematic identifier
+
+Revision ID: b0871ff12f95
+Revises: 14e5db4fde8e
+Create Date: 2017-03-21 12:28:59.518734
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b0871ff12f95'
+down_revision = '14e5db4fde8e'
+
+from alembic import context, op
+import sqlalchemy as sa
+
+
+def upgrade(pyramid_env):
+    with context.begin_transaction():
+        op.add_column(
+            'thematic',
```

Comment:
It's done in https://github.com/assembl/assembl/pull/87/commits/c8b1c038db79ac28a312a7f489bc5b7ac0b12478